File: testresources/csharp/benchmark/Domain/Direct/Violating/CallInstanceInnerClass.cs
namespace Domain.Direct.Violating
{
	using Domain.Direct;
	using Technology.Direct.Dao;

	public class CallInstanceInnerClass
	{
		CallInstanceOuterClassDAO.CallInstanceInnerClassDAO innerDao;

		public int CallMethodInstanceInnerClass()
		{
			return innerDao.GetNext();
		}
	}
}

File: testresources/java/accuracy/src/domain/direct/violating/CallInstanceInnerClass.java
package domain.direct.violating;

import technology.direct.dao.CallInstanceOuterClassDAO;

public class CallInstanceInnerClass {
	
	protected CallInstanceOuterClassDAO.CallInstanceInnerClassDAO innerDao; // Declaration of inner class, while only the outer class is imported

	public int CallMethodInstanceInnerClass() {
		int b;
		b = innerDao.getNext();
		return b;
	}
	
}
